dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/ItemRestRepository.java

```diff
@@ -78,6 +93,19 @@ public ItemRest findOne(Context context, UUID id) {
         return page;
     }
 
+    @Override
+    public void patch(Context context, HttpServletRequest request, String apiCategory, String model, UUID uuid, Patch
+        patch)
+        throws UnprocessableEntityException, PatchBadRequestException, SQLException, AuthorizeException,
+        ResourceNotFoundException {
+
+        ItemRest restModel = findOne(context, uuid);
+        if (restModel == null) {
+            throw new ResourceNotFoundException(apiCategory + "." + model + " with id: " + uuid + " not found");
```

Comment:
Andrea, this does not have the effect that I expected.  The controller's patch method doesn't specifically handle ResourceNotFoundException and instead returns 500.  

I guess to my way of thinking, if the item is not found one might return 422 rather than 404.  However, the controller can be modified to return 404 if that is the correct behavior.

Reply:
I modified to controller so that it throws the ResourceNotFoundException.